connectors/source.py

```diff
@@ -275,7 +279,15 @@ def get_source_klass(fqn):
     return getattr(module, klass_name)
 
 
-def get_data_sources(config):
+def get_source_klasses(config):
     """Returns an iterator of all registered sources."""
     for name, fqn in config["sources"].items():
```

Comment:
Same here: it's not obvious to me what `fqn` means? "fully qualified name"? "frequency"?

Reply:
Yes, `Fully Qualified Name`